packages/core/src/Editor.ts

```diff
@@ -276,7 +284,33 @@ export class Editor extends EventEmitter<EditorEvents> {
    * Creates a ProseMirror view.
    */
   private createView(): void {
-    const doc = createDocument(this.options.content, this.schema, this.options.parseOptions)
+    let doc: ProseMirrorNode
+
+    try {
+      doc = createDocument(
+        this.options.content,
+        this.schema,
+        this.options.parseOptions,
+        { errorOnInvalidContent: this.options.enableContentCheck },
+      )
+    } catch (e) {
+
+      // Remove the collaboration extension if the content is invalid, to not sync invalid content
```

Comment:
I wonder if this is always the desired behavior. In any case, we should document it.

Reply:
+ I wonder if it's breaking because the behavior was different before.

Reply:
I put it there more as a conversation starter, that it is at least possible to do here. I wanted to talk more about what our options should be. Perhaps we can give the developer choice in how they want to handle this with some sort of a callback function to execute this maybe?

Reply:
I like the callback idea. @bdbch what do you think?